sql/Register.sql

```diff
@@ -0,0 +1,12 @@
+create table user_table(
+	username varchar(30)  unique not null,
+	userpassword varchar(30) not null,
+	email varchar(30) unique not null,
+	mobileNo varchar(10) unique,
+	age int,
```

Comment:
add check constraint 
`CHECK( age > 0 and age<= 100 )`

Reply:
Added Constraint to check the age.